src/Telemetry/Telemetry.cs

```diff
@@ -383,6 +383,7 @@ public enum TelemetryMeasureName
         GetPrimaryKeysDurationMs,
         GetUnprocessedChangesDurationMs,
         InsertGlobalStateTableRowDurationMs,
+        MaxBatchSize,
```

Comment:
Can BatchSize in line 375 be removed?

Reply:
That's being used for the AsyncCollector which also has a batch size (but that's hardcoded to 1000 and is a limitation of how many items can be inserted at once)